package.json

```diff
@@ -1466,9 +1504,14 @@
         },
         {
           "command": "cmake.projectStatus.selectBuildPreset",
-          "when": "view == cmake.projectStatus && cmake:enableFullFeatureSet && viewItem == 'buildPreset'",
+          "when": "view == cmake.projectStatus && cmake:enableFullFeatureSet && viewItem =~ /buildPreset/",
```

Comment:
here too for the quotes (single quotes)

Reply:
No. The `=~` operator asks for regex, so the right hand side needs to be regex, you can see the documentation that I followed here: https://code.visualstudio.com/api/references/when-clause-contexts#match-operator